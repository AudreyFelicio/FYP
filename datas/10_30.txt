10 30
3 2 8 6 9 5 10 1 7 4 
8 7 4 5 1 10 2 9 3 6 
2 5 7 6 1 3 4 8 9 10 
8 1 10 5 9 6 3 2 7 4 
3 1 10 9 4 5 2 8 6 7 
4 10 7 6 3 2 1 5 8 9 
1 9 5 4 6 7 3 2 10 8 
4 1 6 9 7 3 8 2 5 10 
3 6 5 2 10 9 8 7 1 4 
9 6 8 2 5 7 4 3 1 10 
5 6 4 8 7 1 2 9 3 10 
7 8 2 10 1 5 3 4 9 6 
3 10 1 8 7 2 9 5 6 4 
5 10 9 1 3 8 4 6 2 7 
6 4 3 2 1 7 9 5 8 10 
7 8 1 10 5 6 2 9 3 4 
7 1 9 10 4 6 3 2 5 8 
3 1 2 9 4 7 6 5 8 10 
5 9 7 3 8 1 10 2 6 4 
6 8 7 4 10 5 1 9 2 3 
6 5 10 7 9 2 3 8 1 4 
3 5 10 4 9 2 6 8 1 7 
9 1 2 6 10 8 7 5 4 3 
3 2 8 9 1 5 7 6 10 4 
8 10 5 7 6 9 1 4 3 2 
9 5 3 8 10 1 2 7 4 6 
6 3 10 4 1 7 5 9 2 8 
7 2 9 1 6 3 4 8 5 10 
7 10 9 8 2 3 1 4 5 6 
9 7 8 2 5 1 3 6 10 4 
