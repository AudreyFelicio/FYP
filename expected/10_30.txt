147
9 6 3 8 2 7 1 5 10 4 
